### PR TITLE
[OSJC-262] Retrieve logs via websocket

### PR DESCRIPTION
**Pull request by jcantrill**
This PR add a new capability
* retrieve logs without binary dependency

### CODE REVIEW

**Comment by jcantrill** [comment]
[test]

cc @fbricon @adietish

File: src/main/java/com/openshift/restclient/http/IHttpConstants.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 	public static final int STATUS_UNAUTHORIZED = 401;
 	public static final int STATUS_FORBIDDEN = 403;
 	public static final int STATUS_NOT_FOUND = 404;
+	
+	public static final int STATUS_NORAMAL_STOP = 1000;
```
Comment (fbricon) [inline src/main/java/com/openshift/restclient/http/IHttpConstants.java]: typo

**Comment by jcantrill** [comment]
[test]  @fbricon fixed type

**Comment by jcantrill** [comment]
@gabemontero might this be of interest to you?  any review? suggestions regarding the api?

**Comment by gabemontero** [comment]
@jcantrill definitely interested - i'll try and take a peek later today or tomorrow

**Comment by gabemontero** [comment]
@jcantrill ok, got a chance to look.  The integration test case you've got spelled a typical sample usage  out clearly.  The API is definitely consumable from my end.... lgtm

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**